fixtures/receivers/README.md

```diff
@@ -0,0 +1,44 @@
+# NFL Receiver Statistics from the 2018 Regular Season
+
+**Downloaded from Pro-Football-Reference on June 18, 2019.**
+
+- Multivariate Data Set
+- 494 Instances
+- 20 attributes
+- Well suited for _clustering_ tasks
+- [https://www.pro-football-reference.com/years/2018/receiving.htm](https://www.pro-football-reference.com/years/2018/receiving.htm)
+
+## Abstract
+
+This dataset is comprised of statistics on all eligible receivers from the 2018 NFL regular season.
+
+## Description
+
+The dataset consists of an aggregate of all relevant statistics for eligible receivers that played in at least 1 game and had at least 1 target throughout the season. This is not limited to players specifically designated as wide-receivers, but may include other positions such as running-backs and tight-ends.
+
+## Attributes
+
+- Player : player name
+- Id : PFR player id
+- Tm : team
+- Age : age
+- Pos : position
+- G : games count
+- GS : games started count
+- Tgt : number of targets
+- Rec : number of receptions
+- Ctch_Rate : catch rate (Rec / Tgt)
+- Yds : total yards
+- Y/R : yards per reception
+- TD : touchdowns
+- Lng : longest catch
+- Y/Tgt : yards per target
+- R/G : receptions per game
+- Y/G : yards per game
+- Fmb : number of fumbles
+- ProBowl : 1/0 indicator of making the Pro Bowl
+- FirstTeamAllPro : 1/0 indicator of making First Team All Pro
+
+## Citation
+
+Sports Reference LLC. "2018 NFL Receiving." Pro-Football-Reference.com - Pro Football Statistics and History. https://www.pro-football-reference.com/06-18-2019
```

Comment:
If possible, I'd like to add a note here that says something to the effect of: "Redistributed with the permission of Sports Reference LLC on [date of email] via email." 

Also, the link isn't working, should the reference look like the following?

Sports Reference LLC, “2018 NFL Receiving,” Pro-Football-Reference.com - Pro Football Statistics and History. [Online]. Available: https://www.pro-football-reference.com/years/2018/receiving.htm. [Accessed: 18-Jun-2019].

Reply:
I'm so happy hear that you got a response from them! I definitely like the idea of documenting it with the brief note that @bbengfort suggested.